tests/php/UsedOnTableTest/UsedOnTableExtension.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace SilverStripe\Admin\Tests\UsedOnTableTest;
+
+use SilverStripe\Assets\Shortcodes\FileLink;
+use SilverStripe\Core\Extension;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\DataObject;
+
+class UsedOnTableExtension extends Extension
+{
+    public function updateUsage(ArrayList &$usage, DataObject &$record)
+    {
+        $usage = $usage->exclude([
+            'ClassName' => [
+                FileLink::class,
```

Comment:
Should try and remove something other than FileLink since that's already removed by default https://github.com/silverstripe/silverstripe-asset-admin/blob/1.6/code/Extensions/UsedOnTableExtension.php

Even though it's on a different module, something may get confused when running this in the context of silverstripe/recipe-cms or kitchen-sink

Try and remove something generic such as a Page

Reply:
@bergice still one unresolved comment here

Reply:
We override the extensions in the tests which should resolve this.

Reply:
What I mean is update the test so that you're excluding File's on Pages from showing in the UsedOnTable, rather than Files associated with FileLink, because FileLink is removed by default anywhere whereas Pages aren't, so this way we're testing that the `updateUsage` extension is actually removing something

Reply:
How come `FileLink`'s are removed by default and where does this happen?

Reply:
In UsedOnTableExtension.php - I linked to that a few comments above

Reply:
That's what I meant by excluding having the extension be excluded in the tests: https://github.com/silverstripe/silverstripe-admin/pull/957/files#diff-1a65f9abdcc3690e7d4512d51a4ac09aR16

Reply:
OK gotcha